lib/improver/threshold.py

```diff
@@ -62,15 +63,15 @@ def __init__(self, threshold, fuzzy_factor=None,
                 False to count points as significant if *above* the threshold.
 
         Raises:
-            ValueError: If a threshold of 0.0 is requested.
             ValueError: If the fuzzy_factor is not greater than 0 and less
                         than 1.
 
         """
-        if threshold == 0.0:
```

Comment:
Is there a reason this was here in the first place? Is it because this doesn't work with the fuzzy thresholding, maybe we should add something in case a threshold is zero and we are trying to multiplicative fuzziness. In this case maybe we should raise a warning?

Reply:
Yeah, I guess that is all it can have been for. I'll add a suitable error and unit test.